File-Formats/DocIO/Assemblies-Required.md

```diff
@@ -140,6 +140,8 @@ Syncfusion.DocIO.Portable<br/>Syncfusion.Compression.Portable<br/>Syncfusion.Off
 Syncfusion.DocIO.NET<br/>Syncfusion.Compression.NET<br/>Syncfusion.OfficeChart.NET<br/>Syncfusion.Pdf.NET<br/>Syncfusion.DocIORenderer.NET<br/>Syncfusion.Pdf.Imaging.NET<br/> SkiaSharp<br/>Syncfusion.SkiaSharpHelper.NET</td></tr>
 </table>
 
+N> Starting with v24.1.x, if you reference "Syncfusion.DocIORenderer", you also have to add "Syncfusion.Pdf.Imaging" assembly reference in your projects to perform Word to Image conversion.
```

Comment:
N> Starting with the v24.1.x, if you reference "Syncfusion.DocIORenderer", you have to add the "Syncfusion.Pdf.Imaging" assembly reference in your projects to perform Word to Image conversion.

Reply:
Modified the sentences